ktorm-support-postgresql/src/main/kotlin/me/liuwj/ktorm/support/postgresql/PostgreSqlDialect.kt

```diff
@@ -47,6 +47,10 @@ open class PostgreSqlFormatter(database: Database, beautifySql: Boolean, indentS
         return result
     }
 
+    override val String.quoted: String get() {
+        return "${database.identifierQuoteString}${this}${database.identifierQuoteString}".trim()
```

Comment:
I am quoting _all_ column names. I believe quoting all columns for consistency is the right approach, but additional logic could be added to only quote column names which contain one or more uppercase characters.